library/headers.inc.php

```diff
@@ -19,6 +19,21 @@
  */
 ?>
 <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+<style type="text/css">
```

Comment:
I am not sure where/when a stylesheet reference crept into this file, but the intention of this file is just a library of asset function calls.  Everything in this file should be a function that calls includes.  Nothing else.
$css_header variable should contain the file location for holding this stuff (I think).

Reply:
We need to make .css files in to .php files to use php variables inside them, so that we can use the colors we choose

Reply:
By adding this feature i didnt want to break anything inside the ui, so i separated the bg classes and echoed out with our php variables

Reply:
I get why you did it.  I am just hoping you can find a better place to put it than this particular file...unless it turns out that it is really really really the best place, in which case it should just have a function to call the code file, not the code.

Reply:
Im planning to remove the <link rel= "stylesheet" and replace it with style.php file, i need to know are we ready to remove themes and adopt the theme setter.

Reply:
Yes...provided we have things that look good, and we have padding control too, then we are set to move forward.  We just can't have the project go into an unusable state during a transition.

Reply:
I will continue it to make a simple single theme with customizable options and make it ready for merge as soon as possible.

Reply:
Sounds great.  Sending you a collaborator invite.